### PR TITLE
refactor: corner selling, use qbx lib instead of utils, ox_lib locales

### PR DESCRIPTION
**Pull request by nitrou5**
## Description

refactor: corner selling, use qbx lib instead of utils and ox_lib locales

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by Manason — APPROVED** [review]
Reviewed corner selling change. Skimmed other files.

File: server/cornerselling.lua [hunk 1]
```diff
@@ -7,14 +7,13 @@ local function getAvailableDrugs(source)
...
             availableDrugs[#availableDrugs + 1] = {
-                item = item.name,
-                amount = item.amount,
-                label = exports.ox_inventory:Items()[item.name].label
+                item = itemName,
+                amount = itemCount,
+                label = exports.ox_inventory:Items()[itemName].label
             }
         end
     end
```
Comment (solareon) [inline server/cornerselling.lua:19]: ox_inventory supports searching with a table per the [documentation](https://overextended.dev/ox_inventory/Functions/Server#search). Could we rewrite this to do one search to the players inventory and then use that result to feed into available drugs. We shouldn't even need a loop in that case.

**Review by solareon — CHANGES_REQUESTED** [review]

File: server/cornerselling.lua (hunk 1 above)
Comment (nitrou5) [reply server/cornerselling.lua:19]: When we provide a table for ox_inventory:Search it returns all the items including the ones non existent in the inventory. 

`{"weed_purple-haze":0,"weed_amnesia":0,"crack_baggy":0,"weed_og-kush":0,"weed_white-widow":7,"weed_ak47":10,"weed_skunk":0,"cokebaggy":11,"meth":0}`

So I think a loop is needed here regardless?

Comment (solareon) [reply server/cornerselling.lua:19]: So we could do that as is on the server side and then when it goes back to the client let it do the random at https://github.com/Qbox-project/qbx_drugs/blob/e68693b8d989fa85228461d0a60cbb56e7e67e94/client/cornerselling.lua#L122-L123 inside of a [lib.WaitFor](https://overextended.dev/ox_lib/Modules/WaitFor/Shared) that checks if the random returns an item with an amount > 0. 

Or better still we just have the server decide what item and amount to sell after the search and send it back to the client just for the notification/offer side

Comment (nitrou5) [reply server/cornerselling.lua:19]: I dont like removing the loop from the server and using lib.WaitFor idea as it simply stresses more the client, correct me if I am wrong. For the server to decide what item/amount is possible but may be with a different server callback? in all cases another PR will follow this week with better suggestion/approach

**Review by solareon — APPROVED** [review]

**Pull request merged**